test/runtime/write.c

```diff
@@ -617,6 +617,67 @@ static void usage(const char *program_name)
            p);
 }
 
+static void fs_stress_test() 
+{
+    int num_files = 1000;
+    int fds[num_files];
+    char *fd_names[num_files];
+
+    /* Creating and writing to new files */
+    int i;
+    for (i = 0; i < num_files; i++) {
+        char buf[BUFLEN];
+        ssize_t rv;
+        char name[25];
+        sprintf(name, "fs_stress_test_%d", i);
+        int fd = open(name, O_CREAT | O_RDWR, S_IRUSR | S_IWUSR);
+        fds[i] = fd;
+        fd_names[i] = name;
+        if (fd < 0) {
+            perror("open");
+            exit(EXIT_FAILURE);
+        }
+
+        _READ(buf, BUFLEN);
+
+        if (rv >= BUFLEN)
+            rv = BUFLEN - 1;
+        buf[rv] = '\0';
```

Comment:
What's the purpose of the 3 lines above?

Reply:
We actually were using these lines with our `writetest_debug` statements, but since we got rid of those, these lines are no longer needed. We deleted these three lines.